mbed/mbed.py

```diff
@@ -307,16 +308,12 @@ class Bld(object):
     name = 'bld'
     default_branch = 'default'
 
-    def isurl(url):
-        m_url = re.match(regex_url_ref, url.strip().replace('\\', '/'))
-        if m_url:
-            return re.match(regex_build_url, m_url.group(1))
-        else:
-            return False
-
     def init(path):
         if not os.path.exists(path):
             os.mkdir(path)
```

Comment:
Silly question. Are we assuming that this will always work (say, in the case where a user attempts to create a directory where they don't have permissions to)?

Reply:
`Bld.init()` is called within `try:` statement. See `Bld.clone()` below.